applications/GeoMechanicsApplication/tests/cpp_tests/custom_retention/test_saturated_below_phreatic_level_law.cpp

```diff
@@ -0,0 +1,113 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+#include "custom_retention/saturated_below_phreatic_level_law.h"
+#include "geo_mechanics_application_variables.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+namespace
+{
+
+using namespace Kratos;
+
+SaturatedBelowPhreaticLevelLaw CreateSaturatedBelowPhreaticLevelLaw()
+{
+    return SaturatedBelowPhreaticLevelLaw{};
+}
+
+} // namespace
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(SaturatedBelowPhreaticLevelLawReturnsCloneOfCorrectType, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    const auto law         = CreateSaturatedBelowPhreaticLevelLaw();
```

Comment:
So, for instance here I would have written:
```suggestion
    const auto law         = SaturatedBelowPhreaticLevelLaw{};
```

I think we can do the same thing in other places.

Reply:
done